docs/compositions.md

```diff
@@ -4,38 +4,44 @@ Action compositions make it possible to dynamically build and invoke a series of
 
 ## Example
 
-Suppose we define an _increment_ action:
+Suppose we define an _increment_ action in a source file `increment.js`:
 
-```
-$ cat > increment.js
+```javascript
 function main({ value }) { return { value: value + 1 } }
-^D
-
-$ wsk action create increment increment.js
 ```
 
-We can use this _increment_ action in a composition as follows:
+We create the action _increment_:
 
 ```
-$ cat > composition.js
+wsk action create increment increment.js
+```
+
+We can use this _increment_ action in a composition. We create a source file `composition.js`:
+
+```javascript
 function main(params) {
     switch (params.$step || 0) {
         case 0: delete params.$step; return { params, action: 'increment', state: { $step: 1 } }
         case 1: delete params.$step; return { params, action: 'increment', state: { $step: 2 } }
         case 2: delete params.$step; return { params }
     }
 }
-^D
+```
+
+We create the composition action:
 
-$ wsk action create composition composition.js -a conductor true
+```
+wsk action create composition composition.js -a conductor true
 ```
 
 The key to making this action a composition is the _conductor_ annotation, which we discuss in the next section.
 
 This example composition executes two _increment_ actions in a sequence:
 
 ```
-$ wsk action invoke composition -br -p value 3
+wsk action invoke composition -br -p value 3
```

Comment:
-r is the same as -br, the -b is implied.

Reply:
done